index.html

```diff
@@ -68,6 +68,165 @@ <h1 class="main__title">I'm Alexandra. <br> Glad to see you!</h1>
+                        </a>
+                    </li>
+                    <li class="about__socialMedia__container"> 
+                        <a href="" class="about__socialMedia__links">
+                            <i class="fa-brands fa-instagram"></i>
+                        </a>
+                    </li>
+                </ul>
+                <button class="about__button">Get my resume</button>
+            </div>
+
+            <div class="about__abilities">
+                <div class="about__abilities__container">
+                    <h2 class="about__abilities__title">Languages</h2>
+                    <figcaption class="abilities__image__container">
+                        <img src="/assets/arrowdown.png" alt="" class="abilities__image">
+                    </figcaption>
+                </div>
+                <ul class="about__list__container">
+                    <li class="about__list__item">
+                        <div class="about__language">
+                            <figcaption class="about__image__container">
+                                <img class="about__image" src="/assets/js.png" alt="javascript">
+                            </figcaption>
+                            <h2 class="about__list__title">JavaScript</h2>
+                        </div>
+                    </li>
+                    <li class="about__list__item">
+                        <div class="about__language">
+                            <figcaption class="about__image__container">
+                                <img src="/assets/html.png" alt="html" class="about__image">
+                            </figcaption>
+                            <h2 class="about__list__title">HTML</h2>
+                        </div>
+                    </li>
+                    <li class="about__list__item">
+                        <div class="about__language">
+                            <figcaption class="about__image-_container">
+                                <img src="/assets/css.png" alt="css" class="about__image">
+                            </figcaption>
+                            <h2 class="about__list__title">CSS</h2>
+                        </div>
+                    </li>
+                </ul>
+                <div class="about__abilities__container active">
+                    <h2 class="about__abilities__title">Frameworks</h2>
+                    <figcaption class="abilities__image__container">
+                        <img src="/assets/arrowright.png" alt="" class="abilities__image">
+                    </figcaption>
+                </div>
+                <div class="about__abilities__container active">
+                    <h2 class="about__abilities__title">Skills</h2>
+                    <figcaption class="abilities__image__container">
+                        <img src="/assets/arrowright.png" alt="" class="abilities__image">
+                    </figcaption>
+                </div>    
+            </div>
```

Comment:
- [ ] Please consider to use `ul` to wrap the list of  The technology in each category ("Languages", "Frameworks" and "Skills") and make sure each of them is `li`. This will make your code way more cleaner 🧼 and improve the user experience 👍